requirements.txt

```diff
@@ -9,3 +9,5 @@ junit-xml>=1.0,<2.0
 lockfile
 six>=1.0,<2.0
 colorama>=0.3,<0.5
+beautifulsoup4
+lxml; sys_platform == 'darwin'
```

Comment:
I think a comment about the installation requirements of `beautifulsoup4` would be useful so that `lxm` is not removed without knowing why it is included with being explicitly imported.